src/test/java/embl/ebi/variation/eva/pipeline/steps/IndicesCreateTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package embl.ebi.variation.eva.pipeline.steps;
+
+import com.mongodb.DBCollection;
+import com.mongodb.MongoClient;
+import embl.ebi.variation.eva.VariantJobsArgs;
+import embl.ebi.variation.eva.pipeline.jobs.InitDBConfiguration;
+import embl.ebi.variation.eva.pipeline.config.InitDBConfig;
+import embl.ebi.variation.eva.pipeline.jobs.JobTestUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.batch.core.BatchStatus;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import embl.ebi.variation.eva.pipeline.steps.tasklet.IndicesCreate;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+/**
+ * Created by jmmut on 2016-08-26.
+ *
+ * @author Jose Miguel Mut Lopez &lt;jmmut@ebi.ac.uk&gt;
+ * Test {@link IndicesCreate}
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = { InitDBConfiguration.class, InitDBConfig.class, JobLauncherTestUtils.class})
+public class IndicesCreateTest {
```

Comment:
A test case should check that no duplicates can be inserted. It doesn't need to use Spring Batch writers, just regular insertions will do.